modules/mobile-agents/windows/jax-rs/src/main/java/org/wso2/carbon/mdm/mobileservices/windows/services/wstep/impl/CertificateEnrollmentServiceImpl.java

```diff
@@ -122,10 +121,10 @@ public void requestSecurityToken(String tokenType, String requestType,
                 headerBinarySecurityToken = element.getFirstChild().getNextSibling().getFirstChild().getTextContent();
             }
         }
-        List<ConfigurationEntry> tenantConfigurations = null;
+        List<ConfigurationEntry> tenantConfigurations;
         try {
-            if (getTenantConfigurationData() != null) {
-                tenantConfigurations = getTenantConfigurationData();
+            if (WindowsAPIUtils.getTenantConfigurationData() != null) {
+                tenantConfigurations = WindowsAPIUtils.getTenantConfigurationData();
```

Comment:
rather than calling WindowsAPIUtils.getTenantConfiguration() two times, better to call it once, get the value to a local variable and then check for null value.